.travis.yml

```diff
@@ -46,9 +46,6 @@ script:
   - bin/test
   - bin/test-plone_addon
 
-after_script:
```

Comment:
@jensens could you elaborate why you think that xvfb might be the problem here? I don't see how the check-readme fail could be related. I'd be ok with just merging this pr to get back to a green master build. If we start to fiddle around in the same branch/pr, that becomes harder...